### PR TITLE
Adds load testing macros

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request
Adds a macro that consumes cpu up to some tick limit
Adds a define that uses said macro to leave verb processing just its reserve of the tick
Adds a second subdefine that forces verb processing into total overtime, mimicing the worst case scenario around maptick spikes

Would like @MrStonedOne's opinion on this, since I am "technically" adding this to the mc's main loop

## Why It's Good For The Game

Basically, one of our main bugaboos right now is the overtime that verbs, particularly clicks, cause.
Because verbs run after literally everything else (See #65589), it is useful to have a way to force them into bad and worst case scenarios
Makes testing this stuff easier, plus we can use the macro to simulate load in other ways too. It's good.

### CODE REVIEW

**Review by MrsTonedOne — APPROVED** [review]

**Comment by WarlockD** [comment]
WHERE THE FUCK WAS THIS FOR 10 YEARS.
